var/spack/repos/builtin/packages/spykfunc/package.py

```diff
@@ -34,9 +34,11 @@ class Spykfunc(PythonPackage):
     homepage = "https://bbpcode.epfl.ch/code/#/admin/projects/building/Spykfunc"
     url      = "ssh://bbpcode.epfl.ch/building/Spykfunc"
 
-    version('develop', git=url, submodules=True, preferred=True)
+    version('develop', git=url, submodules=True)
+    version('0.11.0', git=url, tag='v0.11.0', submodules=True, preferred=True)
 
-    depends_on('highfive', type='build')
+    depends_on('hdf5~mpi')
```

Comment:
Never ever mpi? ;(

Reply:
Never ever. Python will barf to hard :-/…